### PR TITLE
Automate PyPI release on a tag

### PR DESCRIPTION
**Pull request by TG1999**
Signed-off-by: Tushar Goel <tushar.goel.dav@gmail.com>

### CODE REVIEW

File: .github/workflows/pypi-release.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Release uinvers on PyPI
```
Comment (pombredanne) [inline .github/workflows/pypi-release.yml:1]:
```suggestion
name: Release library as a PyPI wheel and sdist on tag
```

File: .github/workflows/pypi-release.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build-n-publish:
+    name: Build and publish univers to PyPI
```
Comment (pombredanne) [inline .github/workflows/pypi-release.yml:9]:
```suggestion
    name: Build and publish library to PyPI
```

File: .github/workflows/pypi-release.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    runs-on: ubuntu-20.04
+
+steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
```
Comment (pombredanne) [inline .github/workflows/pypi-release.yml:17]:
```suggestion
        python-version: 3.8
```

File: .github/workflows/pypi-release.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    types: [created]
+
+jobs:
+  build-n-publish:
+    name: Build and publish univers to PyPI
+    runs-on: ubuntu-20.04
+
+steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.6
```
Comment (pombredanne) [inline .github/workflows/pypi-release.yml:14]:
```suggestion
    - name: Set up Python
```

**Comment by TG1999** [comment]
https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

File: .github/workflows/pypi-release.yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: >-
+        python -m
+        build
+        --sdist
+        --wheel
+        --outdir dist/
+        .
+    - name: Publish distribution 📦 to PyPI
```
Comment (pombredanne) [inline .github/workflows/pypi-release.yml:32]:
```suggestion
    - name: Publish distribution  to PyPI
```

File: .github/workflows/pypi-release.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      if: startsWith(github.ref, 'refs/tags')
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        password: ${{ secrets.PYPI_API_TOKEN }}
```
Comment (pombredanne) [inline .github/workflows/pypi-release.yml:36]:
```suggestion
        password: ${{ secrets.PYPI_API_TOKEN }}
        
```

**Review by pombredanne — APPROVED** [review]
Here are a few tiny nits...Thanks

**Review by pombredanne — APPROVED** [review]

**Pull request merged**